index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <link rel="stylesheet" href="style.css">
```

Comment:
Your HTML links to your styles, but does not incorporate a webfont, which is one of the requirements for [Project 1](https://mica-web.github.io/gd-220/projects/project01/#requirements).